mobile/src/main/java/ch/epfl/sdp/mobile/application/authentication/AuthenticatedUser.kt

```diff
@@ -83,7 +85,7 @@ class AuthenticatedUser(
    * @param puzzle the [Puzzle] to mark as solved.
    */
   suspend fun solvePuzzle(puzzle: Puzzle) {
-    firestore.collection("users").document(this.uid).update {
+    firestore.collection(ProfileDocument.Collection).document(this.uid).update {
       arrayUnion("solvedPuzzles", puzzle.uid)
```

Comment:
Missing field.

Reply:
Added in 715a39794e4fd9228eaaf11ad0bf8a1682f5edfd